vach/src/loader/archive.rs

```diff
@@ -88,11 +91,13 @@ impl<T: Seek + Read> Archive<T> {
 				// The ID is part of the signature, this prevents redirects d by mangling the registry entry
 				buffer.extend(id.as_bytes());
 
-				if let Err(error) = pub_key.verify(&buffer, &entry.signature) {
-					anyhow::bail!(format!(
-						"({}): Invalid signature found for leaf with ID: {}",
-						error, id
-					))
+				// If there is an error the data is flagged as invalid
+				if entry.signature.is_some() {
+					if let Err(info) = pub_key.verify(&buffer, &entry.signature.unwrap()) {
+						 eprintln!("{}", &info);
```

Comment:
Shouldn't we use log::error or log::warn for this, so that we don't forcefully print something?